DockerfileTest

```diff
@@ -0,0 +1,25 @@
+# syntax=docker/dockerfile:1
```

Comment:
I'd rather stay with the "official" Dockerfile. I understand you might have used this one to test, but just don't commit this please